zboss/doc/zboss_configuration.rst

```diff
@@ -8,18 +8,41 @@ ZBOSS library configuration
    :depth: 2
 
 The ZBOSS stack is distributed as a set of precompiled linkable libraries.
+These libraries are available in the following versions:
 
-The libraries support different Zigbee device roles, with each variant having its own configuration.
+* `ZBOSS production libraries`_
+* `ZBOSS development libraries`_
+
+Each version supports different Zigbee device roles, with each variant having its own configuration.
 This allows you to scale the application and select the most suitable set of features.
 
+These libraries are used in the Zigbee protocol configuration in the |NCS| when defining the Zigbee device role, as described in :ref:`nrf:zigbee_ug_configuration` in the |NCS| documentation.
+They are included by the OSIF subsystem, which acts as the linking layer between the ZBOSS stack and |NCS|.
+OSIF implements a series of functions used by ZBOSS and is included in the |NCS|'s Zigbee subsystem.
+
+ZBOSS production libraries
+**************************
+
 For a complete list of the ZBOSS configuration options, see the following files:
 
-* :file:`zboss/include/osif/libzboss_config.h` - library for Coordinators and Routers
-* :file:`zboss/include/osif/libzboss_config.ed.h` - library for End Devices
+* :file:`zboss/production/include/osif/libzboss_config.h` - library for Coordinators and Routers
+* :file:`zboss/production/include/osif/libzboss_config.ed.h` - library for End Devices
 
-These libraries are used in the Zigbee protocol configuration in the |NCS| when defining the Zigbee device role, as described in :ref:`nrf:zigbee_ug_configuration` in the |NCS| documentation.
-They are included in the OSIF subsystem, which acts as the linking layer between the ZBOSS stack and |NCS|.
-OSIF implements a series of functions used by ZBOSS and is included in the |NCS|'s Zigbee subsystem.
+The ZBOSS production library version is enabled by default with the :kconfig:`CONFIG_ZIGBEE_LIBRARY_PRODUCTION` Kconfig option.
+
+ZBOSS development libraries
+***************************
+
+The ZBOSS libraries in the development state include all the production code, but also features that are still in the experimental state.
+
+For a complete list of the ZBOSS configuration options, see the following files:
+
+* :file:`zboss/development/include/osif/libzboss_config.h` - library for Coordinators and Routers
+* :file:`zboss/development/include/osif/libzboss_config.ed.h` - library for End Devices
+
+You can select the ZBOSS development library version with the :kconfig:`CONFIG_ZIGBEE_LIBRARY_DEVELOPMENT` Kconfig option.
+Those libraries include implementation of the eight version of the ZCL specification.
+This implementation includes new features, defined in the new version of the specification, but they introduce incompatibilities with devices, that implement older versions of the specification.
```

Comment:
Please remove the first and third comma, so it reads as:

"This implementation includes new features defined in the new version of the specification, but they introduce incompatibilities with devices that implement older versions of the specification."

Reply:
And perhaps this should be in a "Note" box? (See [this link on Note sections](https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Style+and+usage+guide)

If you agree, then the file should be edited like so:
`.. note:: ` and the text you want to include in the note box starts on the next line, tabbed (three spaces) in.